runners/phpcs/script.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# @file
+# phpcs integration - Script step.
+
+set -e $DRUPAL_TI_DEBUG
+
+ARGS=( $DRUPAL_TI_PHPCS_ARGS )
+
+$HOME/.composer/vendor/bin/phpcs "${ARGS[@]}"
```

Comment:
What about also using phpcsp (Code Sniffer Drupal Practice) ?

```
ARGS=( $DRUPAL_TI_PHPCSP_ARGS )
$HOME/.composer/vendor/bin/phpcsp "${ARGS[@]}"
```

Reply:
I'm not familiar with the `phpcsp` binary. Is there something I can read to learn about it?

I made a couple of modifications to provide default arguments and to make it easier to invoke `drupal/coder` standards.